src/DIRAC/Core/Utilities/File.py

```diff
@@ -274,7 +274,7 @@ def secureOpenForWrite(filename=None, *, text=True):
     else:
         fd, filename = tempfile.mkstemp(text=text)
     with open(fd, "w" if text else "wb", encoding="ascii") as fd:
-        yield fd
+        yield fd, filename
```

Comment:
Why? I haven't managed to figure out where this is needed

Reply:
This is needed for example by ProxyFile.py